server/utils/comments/comments.util.js

```diff
@@ -42,11 +42,10 @@ class commentValidation {
 
     const userComment = await Comment.findOne({
       where: {
-        userId,
         id
       }
     });
-    if (!userComment) {
+    if (userId !== userComment.userId) {
       return Response(res, 400, 'Only the user of this comment can edit comment');
```

Comment:
Good job @geekdanitek, Kindly correct this response message and make it more descriptive. You might as well look into the test cases to correct this if it exists there.

Reply:
Ok @samorahno this will be implemented.

Reply:
@samorahno Feedback implemented.